### PR TITLE
Add label style to configure the styling of a Label dynamically

### PR DESCRIPTION
**Pull request by mrylmz**

### CODE REVIEW

File: Sources/Component/Label/LabelStyle.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+/// A data structure describing the styles of a `Label`.
+public struct LabelStyle {
+    /// The color used to render the text of a `Label`.
+    public let color: UIColor
+
+    /// The font used to render the text of a `Label`.
+    public let font: UIFont
+
+    /// The number of lines the `Label` is allowed to line break.
+    public let numberOfLines: Int
```
Comment (JensK611) [inline Sources/Component/Label/LabelStyle.swift:13]: If you provide the numberOfLines you should also give the possiblity to adjust lineBreakMode and textAlignment for completeness. Usually these three properties are used together.

File: Sources/Component/Label/LabelStyle.swift [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    /// The designated initializer of `LabelStyle`.
+    ///
+    /// - Parameters:
+    ///   - color: The color used to render the text of a `Label`.
+    ///   - font: The font used to render the text of a `Label`.
+    ///   - numberOfLines: The number of lines the `Label` is allowed to line break.
+    public init(
+        color: UIColor,
```
Comment (JensK611) [inline Sources/Component/Label/LabelStyle.swift]: I suggest to provide the default values here to make the parameters optional.

**Review by JensK611 — COMMENTED** [review]
I would appreciate that we cover jamitFoundation with tests.

File: Sources/Component/Label/LabelStyle.swift (hunk 2 above)
Comment (mrylmz) [reply Sources/Component/Label/LabelStyle.swift]: Yes this was a mistake thanks!

**Pull request merged**